zulip/integrations/twitter/twitter-bot

```diff
@@ -258,15 +268,33 @@ for status in statuses[::-1][:opts.limit_tweets]:
         subject = search_term_used
     elif opts.twitter_name:
         subject = composed
-
-    message = {
-        "type": "stream",
-        "to": [opts.stream],
-        "subject": subject,
-        "content": url
-    }
-
-    ret = client.send_message(message)
+    # If we are editing message it implies the current tweet is a retweet
+    # and has been posted before. Hence try to edit the older tweet and
+    # change its number of retweets but if it has passed its edit time
+    # do nothing.
+    if edit_message is True:
+        message_data = {
+            "message_id": message_id,
```

Comment:
`message_id` exists only when the `try` code above doesn't raise an exception, which means it only exists within that scope. Shouldn't `message_id` be initialized to have a default value of 0 first?

Reply:
Added accordingly.